python/ray/serve/utils.py

```diff
@@ -102,9 +107,69 @@ def default(self, o):  # pylint: disable=E0202
         return super().default(o)
 
 
+class ServeHandleEncoder(json.JSONEncoder):
+    """JSON encoder for RayServeHandle and RayServeSyncHandle. Use to enforce
+    JSON serialization of deployment init args & kwargs to faciliate serve
+    pipeline deployment as well as operationaling serve.
+    """
+
+    def default(self, obj):
+        # Import RayServeHandle in utils file lead to import errors
```

Comment:
unit tests for these please

Reply:
I think the simplest thing is to extend our existing `test_handles.py` to do a full round of JSON serde and ensure the outcome is still the same. I've updated two existing tests, lmk if you have other good ones in mind for better coverage

Reply:
yep that sounds good